alembic/versions/2021121222_rm_storage_type_enum_a325a0756797.py

```diff
@@ -0,0 +1,28 @@
+"""rm storage type enum
+
+Revision ID: a325a0756797
+Revises: 685cb4bc0b79
+Create Date: 2021-12-12 22:04:38.060506
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'a325a0756797'
+down_revision = '685cb4bc0b79'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('bundle_store', 'storage_type', type_=sa.String(255), existing_type=sa.Enum("disk", "azure_blob"), nullable=True)
```

Comment:
This is a one-time change. Realized that it's difficult to modify enums so I changed the columns to just be Text types instead, so we don't need migrations for future expansions of supporting new storage types.